.gitignore

```diff
@@ -30,5 +30,6 @@ bin/results
 
 # Some more
 lib/exports.list
-lib/pg_bulkload.sql
+lib/*.sql
```

Comment:
Is it better to change `lib/pg_bulkload--*.sql` ?

Reply:
Yes, It's better than previous. So I changed it.